.github/workflows/main.yml

```diff
@@ -46,13 +46,13 @@ jobs:
             qt-version: "PySide2~=5.15.0"
           - python-version: "3.10"
             qt-lib: "pyqt"
-            qt-version: "PyQt6~=6.2.0 PyQt6-Qt6~=6.2.0"
+            qt-version: "PyQt6==6.2.0 PyQt6-Qt6==6.2.0"
```

Comment:
why this change, I'm fairly certain that `~=` is what I want in this case, I want to get the latest patch version of PyQt6 6.2